spec/features/qing_group_form_spec.rb

```diff
@@ -53,4 +53,50 @@
     within(".form-items") { expect(page).to have_content("New Group Name") }
   end
 
+  scenario "add nested groups to a form" do
+    outer_name = "Outer Group"
+    middle_name = "Middle Group"
+    inner_name = "Inner Group"
+
+    visit(edit_form_path(form, locale: "en", mode: "m", mission_name: get_mission.compact_name))
+    create_group(outer_name)
+    create_group(middle_name)
+    create_group(inner_name)
+
+    click_button("Save") # save form
+
+    # find new parent group's li
+    # append an <ol class="item-list ui-sortable"> to parent
+    # get child li
+    # append child li to new ol
+    # trigger drop_happened on child li with null event
+    execute_script("
+      var new_parent = $('li:contains(\"#{outer_name}\")')[0];
```

Comment:
I tried making a function for this so I could just pass in the parent and child groups names, but I couldn't get it to work - the inner group kept disappearing.

Reply:
Tested a couple times in console, couldn't figure out why it was working there and not in the spec.